editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -1064,21 +1064,23 @@ void Node3DEditorViewport::_select_region() {
 		if (found_nodes.has(sp)) {
 			continue;
 		}
-
 		found_nodes.insert(sp);
 
 		Node *node = Object::cast_to<Node>(sp);
+
+		// Selection requires node is or is descended from edited scene, and has an owner
 		if (node != edited_scene) {
+			if (!node->get_owner() || !edited_scene->is_ancestor_of(node)) {
+				continue;
+			}
```

Comment:
* Checking `is_ancestor_of` fixes both the error message (affecting 4.2.2-stable) and crash (affecting 4.3) identified in #95356
* Skipping on `node != edited_scene && !get_owner` is consistent with 2D https://github.com/godotengine/godot/blob/8e2141eac534f6984bb0bdbcefbd17de27ae0993/editor/plugins/canvas_item_editor_plugin.cpp#L706